blueprint/core/blueprint_ScrollView.h

```diff
@@ -23,25 +23,70 @@ namespace blueprint
     class ScrollView : public View
     {
     public:
+        //==============================================================================
+        // Props following CSS Scrollbars spec with some omissions/additions.
+        // https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Scrollbars
+        static inline juce::Identifier overflowProp       = "overflow";
+        static inline juce::Identifier overflowXProp      = "overflow-x";
+        static inline juce::Identifier overflowYProp      = "overflow-y";
+        static inline juce::Identifier scrollBarWidthProp = "scrollbar-width";
+        static inline juce::Identifier scrollbarColorProp = "scrollbar-color";
+        static inline juce::Identifier scrollOnDragProp   = "scroll-on-drag";
+
         //==============================================================================
         ScrollView()
         {
             addAndMakeVisible(viewport);
-            // TODO: Set these based on props...
-            viewport.setScrollBarsShown(true, false);
+
+            //TODO: Make scrollbars hidden by default so that
+            //      explicit overflow settings required? Or show on
+            //      default and overflow: hidden can be applied?
+            viewport.setScrollBarsShown(false, false);
```

Comment:
I think visible by default, and let `overflow: hidden` if user desires. I believe that matches web behavior